pkgs/development/python-modules/craft-providers/default.nix

```diff
@@ -46,7 +46,7 @@ buildPythonPackage rec {
     # The urllib3 incompat: https://github.com/msabramo/requests-unixsocket/pull/69
     # This is already patched in nixpkgs.
     substituteInPlace pyproject.toml \
-      --replace-fail "setuptools==67.8.0" "setuptools" \
+      --replace-fail "setuptools==69.1.1" "setuptools" \
```

Comment:
Why not use the relaxDeps hook?

Reply:
This doesn't seem to work with pyproject-only projects, or at least I can't make it work